### ISSUE TITLE
Unexpected identifier

### ISSUE THREAD

**Issue by qburst-shanavas** [indutny/node-ip#115]
I am getting this error in node version 6.6. could you please check?

const numberOfAddresses = 2 ** (32 - maskLength);
                               ^

SyntaxError: Unexpected token *
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:549:28)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)

**Comment by adamschofield-nib** [comment]
Also began having this issue with Node 6.# builds since the 1.1.6 release.

**Comment by qburst-shanavas** [comment]
@indutny could you please check?

**Comment by neeths92** [comment]
Facing same issue.
const numberOfAddresses = 2 ** (32 - maskLength);
                               ^
SyntaxError: Unexpected token *
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:549:28)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)

**Comment by wparam** [comment]
Facing same issue here.

**Comment by indutny** [comment]
Sorry about that. Node.js 6 was released 3 years ago so I thought it was time to drop support for it. Perhaps, a better call would have been to publish this as a major version update of the library. Thus, I've just published 1.1.7 which contains a partial revert for the changes that I made in 1.1.6, and it should hopefully work on Node.js 6.

Let me know if it fixes it for you!

**Comment by rajeev-source-code** [comment]
My production build is failing because of this error. Its so hard to find which of the library is using this supporting lib.

**Comment by fengmk2** [comment]
@indutny Still not work in node v6

```
node/v6.16.0/lib/node_modules/foo/node_modules/ip/lib/ip.js:204
var numberOfAddresses = 2 ** (32 - maskLength);
^
SyntaxError: Unexpected token *
at createScript (vm.js:56:10)
```

**Comment by indutny** [comment]
Oops. Missed that! Should be fixed by 8e6f28b. Thanks!

**Comment by neeths92** [comment]
> Oops. Missed that! Should be fixed by [8e6f28b](https://github.com/indutny/node-ip/commit/8e6f28b23a89245eeac74ad2ef9eb0d27cb27e1c). Thanks!

Its working now. Thank you.
